salesagility/SuiteCRM
# Primary Email property is kept after adding an Email address field

Closes #9271 

## Description
As the Issue mentions, the primary email property is lost after adding a new email address. 

This PR adds the code that will make sure that the primary address property is kept after adding new ones.

## Motivation and Context
the primary email address shouldn't be lost.

## How To Test This
1. Create a record in Contacts module with an Email address set as Primary
2. Add a second email
3. Check that the primary property is kept

## Types of changes

### Final checklist
